builtin/common/misc_helpers.lua

```diff
@@ -312,8 +312,7 @@ function core.formspec_escape(text)
 	return text
 end
 
-
-function core.wrap_text(text, max_length, as_table)
+function core.linebreaker function(text, max_length)
```

Comment:
:question:

Reply:
Code tested :100: / :100: 

---
If you want to keep this function public (`minetest`, `core` table), then please document it.